lib/Crypto/extensionplugins.h

```diff
@@ -58,6 +58,12 @@ class SAILFISH_CRYPTO_API CryptoPlugin
     virtual QMap<Sailfish::Crypto::CryptoManager::Algorithm, QVector<Sailfish::Crypto::CryptoManager::KeyDerivationFunction> > supportedKeyDerivationFunctions() const = 0;
     virtual QMap<Sailfish::Crypto::CryptoManager::Algorithm, Sailfish::Crypto::CryptoManager::Operations> supportedOperations() const = 0;
 
+    virtual bool supportsLocking() const;
+    virtual bool isLocked();
```

Comment:
Shouldn't this also be `const`?

Reply:
Done.  Previously it wasn't, because performing database select queries previously was non-const (due to preparation modifying the prepared query cache), however I've now made the cache mutable in order to make those methods const, and thus have now made the isLocked() methods const also.